index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+</head>
+<body>
+  
+    <header class="header">
+      <a href="welcome" class="head">Welcome</a>
+      <nav class="menu-bar">
+        <img src="images/Normal Button/Enabled.png" alt="">
+      </nav>
+    </header>
+    <section class="head-section">
+    <div class="hey-there">
+<h1 class="hey-text">Hey there. I’m Mariane
+ <span class="hey-text2"> I’m a software developer</span></h1>
+    </div>
+    <div class="text1">
+      <p class="text">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </div>
+      <ul class="icons"> 
+        <li>
+          <a href="" class="icon"> <img src="images/icons/twitter.svg" alt=""></a>  
+        </li>  
+         <li>
+          <a href="" class="icon"> <img src="images/icons/i.png" alt="">
+            <img src="images/icons/n.png" alt="">
+           </a> 
+         </li>
+        <li>
+          <a href="" class="icon">  <img  class="medium" src="images/icons/medium@2x.png" alt=""></a>
+        </li>
+        <li>
+          <a href="" class="icon"><img src="images/icons/github.svg" alt=""></a>
+        </li>
+        <li>
+          <a href="" class="icon"><img src="images/icons/angellist.svg" alt=""></a>
+        </li>
+         
+      </ul>
+      <div class="scroll">
+        <img src="images/Normal Button/Disabled.png" alt="">
```

Comment:
- [ ] [OPTIONAL]  The `alt` attribute specifies an alternate text for an image if the image cannot be displayed. The alt attribute provides alternative information for an image if a user for some reason cannot view it (because of a slow connection, an error in the src attribute, or if the user uses a screen reader) ☺️ . Kindly ensure that each `img alt` has the descriptive `alt text` 👍 .